hazelcast/src/main/java/com/hazelcast/query/impl/Indexes.java

```diff
@@ -140,12 +148,12 @@ public static void markPartitionAsUnindexed(int partitionId, InternalIndex[] ind
      * @return new builder instance which will be used to create Indexes object.
      * @see IndexCopyBehavior
      */
-    public static Builder newBuilder(SerializationService ss, IndexCopyBehavior indexCopyBehavior,
+    public static Builder newBuilder(Node node, String mapName, SerializationService ss, IndexCopyBehavior indexCopyBehavior,
                                      InMemoryFormat inMemoryFormat) {
-        return new Builder(ss, indexCopyBehavior, inMemoryFormat);
+        return new Builder(node, mapName, ss, indexCopyBehavior, inMemoryFormat);
     }
 
-    public synchronized InternalIndex addOrGetIndex(IndexConfig indexConfig) {
+    public synchronized InternalIndex addOrGetIndex(String mapName, IndexConfig indexConfig) {
```

Comment:
Why `mapName` passed as an argument here if it's already provided in the constructor and stored in the field?

Reply:
Addressed in https://github.com/hazelcast/hazelcast/pull/21826/commits/d24414371c283311403be5e34da180b89170e119